bot/core/paginator.py

```diff
@@ -0,0 +1,421 @@
+import asyncio
+import logging
+import typing as t
+from textwrap import dedent
+
+import discord
+from discord import Embed
+from discord.abc import Messageable
+from discord.ext import commands
+from discord.ext.commands import Context, Paginator
+
+logger = logging.getLogger(__name__)
```

Comment:
We shouldn't use logging before #83 I know it's my PR and I'm working on it, but we shouldn't touch logging until that, either convert these logs to print statements or wait for #83 to get merged.

Note that even after merging it, this would still be invalid as we'll be using `loguru` for logging, not the default `logging` as is done here